lib/inferno/apps/cli/templates/lib/%library_name%/metadata.rb.tt

```diff
@@ -0,0 +1,18 @@
+require_relative 'version'
+
+module <%= module_name %>
+  class Metadata < Inferno::TestKit
```

Comment:
Can we move the `class Metadata` (or whatever we call it) into lib/%library_name%.rb.tt (i.e: lib/my_test_kit.rb)? And then the content currently in lib/my_test_kit.rb should be moved into lib/my_test_kit/my_test_kit_suite.rb. Then people can copy-paste the suite file to kick start Test Kits that house multiple Test Suites like [PDex](https://github.com/inferno-framework/davinci-pdex-test-kit/blob/main/lib/davinci_pdex_test_kit.rb).

Reply:
I do think we should move the current contents into a separate suite file, but I don't think we should move the metadata there.

Reply:
so `lib/my_test_kit.rb` will only have a couple of includes in it?

I don't care that much one way or another, but Shaumik's suggestion sounds pretty reasonable to me.  This way `lib/my_test_kit.rb` contains metadata about the test kit in a very obvious file (the 'most important' file), and we have fewer files to deal with in the main gem directory?  I can see if you wanted to make it so you can load just the Test Kit metadata without loading everything "within" the test kit too.  But is there a use case for that?  Unfortunately we can't reference this thing in our .gemspec, which means it can't  be used like version.rb is used and why its great to have that in its own file.

Reply:
Yeah, my goal was to be able to load just the test kits without loading the suites, but I guess the platform will need the suites anyway.